### ISSUE TITLE
ENH: Add an option to keep the original aspect in `plot` method when ax!=None

### ISSUE THREAD

**Issue by gepcel** [geopandas/geopandas#1489]
As of pr #1290 `GeoSeries.plot()` will change the aspect of ax if `GeoSeris.crs.is_geographic` by default. If I pass an already existed ax to the `.plot()` method, and want to keep the aspect, I need to `get_aspect` before the plot, and `set_aspect` back after the plot. I can understand this pr and why making it default. But there are still circumstances that the original aspect of the passing ax means something. For example:

```
import geopandas as gpd
import matplotlib.pyplot as plt
from shapely.geometry import Point

gs = gpd.GeoSeries([Point(20, 50), Point(160, 50), Point(160, 50)], crs='epsg:4236')

fig, axs = plt.subplots(1, 2, figsize=(6, 3), sharey=True, 
                        subplot_kw={'projection': ccrs.PlateCarree()})
[ax.stock_img() for ax in axs]
[ax.set_yticks([-50, 0, 50]) for ax in axs]
[ax.text(.01, .99, f'ax{i+1}', ha='left', va='top',
             transform=ax.transAxes) for i, ax in enumerate(axs)]
axs[0].scatter([20, 160], [50, 50], fc='red', transform=ccrs.PlateCarree())
gs.plot(ax=axs[1], fc='red')
plt.show()
```

![image](https://user-images.githubusercontent.com/4179106/85727251-71434e80-b729-11ea-9d7c-32b84ee69b65.png)

To keep the example as simple as possible, it only affects the `sharey` here. One can always argue that `ax2` has better aspect and better layout than `ax1`. The point is if the ax is created by the `plot` method, then `geopandas` can set the aspect to whatever suitable for `geopandas`. But if the ax already exists for some reason other than just `plot.subplot()`, then there should be **at least** an option to keep the aspect untouched.

**Comment by martinfleis** [comment]
In this example, if you pass `aspect='equal'` you get the result you are looking for (i.e. the behaviour of the previous release of geopandas).

However, we can still have an option to make sure we do not touch the `aspect` of an `ax` at all. Still not sure about the use case for that.

**Comment by martinfleis** [comment]
I have found a use case when it is useful not to touch aspect. Compare the following plots. The figure always gets the axis based on the last gdf. In this case, one may want to specify the aspect in the first plot and do not touch it afterwards.

```py
df = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))

ax = df[df['name'] == 'Canada'].plot()
df[df['name'] == 'Bolivia'].plot(ax=ax)
```
![image](https://user-images.githubusercontent.com/36797143/85944078-2d6c6700-b92c-11ea-8ba5-2a48e348e730.png)

```py

ax = df[df['name'] == 'Bolivia'].plot()
df[df['name'] == 'Canada'].plot(ax=ax)
```
![image](https://user-images.githubusercontent.com/36797143/85944081-365d3880-b92c-11ea-9ad6-a0465066981b.png)

So I would add an option to pass `aspect=None`, which will skip the following lines (+ the same for `plot_series`).

https://github.com/geopandas/geopandas/blob/b5ed1a41782220c9c8704e1cbae6636871027b4e/geopandas/plotting.py#L591-L601

@gepcel Would you like to make this change?

**Comment by gepcel** [comment]
Maybe another example:

```
import geopandas as gpd
import matplotlib.pyplot as plt
from shapely.geometry import Point
import cartopy.crs as ccrs

extent=[80, 140, -60, 60]
fig, [ax1, ax2] = plt.subplots(1, 2, figsize=(3, 4), sharey=True,
                               subplot_kw={'projection': ccrs.PlateCarree()})
ax1.stock_img()
ax1.set_extent(extent)
ax1.set_yticks([-50, -20, 0, 20, 50])
gs1 = gpd.GeoSeries([Point(110, -50), Point(110, 50)], crs='epsg:4326')
gs1.plot(ax=ax1)

ax2.stock_img()
ax2.set_extent(extent)
gs2 = gpd.GeoSeries([Point(100, 50), Point(120, 50)], crs='epsg:4326')
gs2.plot(ax=ax2)

plt.show()
```

![a](https://user-images.githubusercontent.com/4179106/85972971-f9647500-ba03-11ea-8b23-e3e7f3590492.png)

**Comment by gepcel** [comment]
@martinfleis I made up a pr #1512, need your help to see if it's ok.

**Comment by jorisvandenbossche** [comment]
Closed by  #1512
